gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsFragment.java

```diff
@@ -480,6 +480,23 @@ public void setMapFeatures(ImmutableSet<MapFeature> features) {
     }
   }
 
+  @Override
+  public void refreshMarkers() {
+    Set<MapPin> mapPinsToAdd = new HashSet<>();
+    for(Marker marker: markers) {
+      MapPin mapPin = (MapPin) marker.getTag();
+
+      if (mapPin != null) {
+        mapPinsToAdd.add(mapPin);
+        removeMarker(marker);
+      }
+    }
+
+    for(MapPin mapPin: mapPinsToAdd) {
```

Comment:
Nit: space after `for` here and above. Please use `google-java-format` plugin to format all changed files.

Reply:
Done